EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
-QUEST_LV_0300_20160310_001287	What? {nl}So the villagers think we ran away after abondoing the monastery?{nl}
-QUEST_LV_0300_20160310_001288	The guys at the religious body.. {nl}They locked people and played both sides against the middle..{nl}
+QUEST_LV_0300_20160310_001287	What? {nl}So the villagers think we ran away after abandoning the monastery?{nl}
+QUEST_LV_0300_20160310_001288	Those people at the Tree of Truth sect... {nl}They locked others inside and alienated them...{nl}
 QUEST_LV_0300_20160310_001289	Can you go to the villagers with me?{nl}If you are with me, all misunderstandings will be solved.
-QUEST_LV_0300_20160310_001290	Hmm.. It seems that the villagers found out the identity of Dominicas.{nl}It is fortunate that the task was solved easier than expected..{nl}
-QUEST_LV_0300_20160310_001291	I will send the villagers back to the village.{nl}I really thank you that you've found Elgos Monastery back from the religious body of the truthful tree.
-QUEST_LV_0300_20160310_001292	The demons here are very strong.{n}Be careful.
-QUEST_LV_0300_20160310_001293	Do not underestimate the power of the vicious idol.{nl}Be careful.
-QUEST_LV_0300_20160310_001294	Now, the revelator would continue the journey by following the will of the goddess.{nl}Before you leave, can you listen to one of my requests?{nl}
-QUEST_LV_0300_20160310_001295	Although I eliminated the religious body of the truthful tree..{nl}The evil demons are still fluttering.{nl}
-QUEST_LV_0300_20160310_001296	If you lend me your power, we will be able to drive out the demons more quickly.{nl}Can you defeat the demons here?{nl}
-QUEST_LV_0300_20160310_001297	There's something that is bothering my mind.{nl}Although, I think driving out the religious body of the truthful tree was fortunate..{nl}
-QUEST_LV_0300_20160310_001298	I mean Dominicas who ran away.{nl}I think she may come back by leading the religious body of the truthful tree.{nl}
-QUEST_LV_0300_20160310_001299	I want to prepare something for this..{nl}Can you help me?
-QUEST_LV_0300_20160310_001300	If Dominicas comes back again..{nl}Then, I would not just stand still.
+QUEST_LV_0300_20160310_001290	Hmm... It seems that the villagers found out the true identity of Dominikas.{nl}It is fortunate that the task was solved easier than expected...{nl}
+QUEST_LV_0300_20160310_001291	I will send the villagers back to the village.{nl}I really appreciate that you've reclaimed the Elgos Monastery from the Tree of Truth sect.
+QUEST_LV_0300_20160310_001292	The demons here are very strong. {nl}Be careful.
+QUEST_LV_0300_20160310_001293	Don't underestimate the power of the sinister idol.{nl}Be careful.
+QUEST_LV_0300_20160310_001294	Now, the Revelator will continue the journey by following the will of the goddess.{nl}But before you leave, can you do me one more favor for this old man?{nl}
+QUEST_LV_0300_20160310_001295	Although I got rid of the Tree of Truth sect...{nl}The evil demons are still fluttering.{nl}
```

Comment:
Not sure what mobs are being referenced in this context, but if they all aren't flying mobs, I feel like "thriving" works better than "fluttering."